.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - lua5.1
+    - luarocks
+
+install:
+  - luarocks --local install luafilesystem
+  - luarocks --local install luacheck
+  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc --no-check-certificate https://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Server/.luacheckrc
```

Comment:
LuaCheck will run the file `.luacheckrc` and this file contains code that will load the file `.luacheckrc_plugin` if existent. Tables in both files will be merged and and options added or overwritten.

Ref.: [main_APIDump.lua](https://github.com/cuberite/cuberite/blob/7922e6addb06de89cc73c64d30321aa6710e30ce/Server/Plugins/APIDump/main_APIDump.lua#L1705)
Build run: https://travis-ci.org/Seadragon91/WorldEdit/builds/240742891